### PR TITLE
Milestone 2 components

### PR DESCRIPTION
**Pull request by Torres-ssf**

### CODE REVIEW

**Comment by Torres-ssf** [comment]
The Microverse Requirements instructions ask to make app.js a stateful component. But the stickler complains about it and aks to make stateless instead

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+const ButtonPanel = () => (
+  <div>
+    <Button buttonName="AC, +/-, %, ÷" />
+    <div>7, 8, 9, X</div>
+    <div>4, 5, 6, -</div>
+    <div>1, 2, 3, +</div>
+    <div>0, ., =</div>
+  </div>
+);
```
[the remark this answers is not in the document]
Comment (Torres-ssf) [reply src/components/ButtonPanel.js]: Screenshot from 2019-12-03 14-06-59
Yes but this was also requested by the instructions. I know that they are supposed to be buttons, but the second milestone specifically asks to divide into groups.

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+const Display = (props) => {
+  const { result } = props;
+  return <p>{result}</p>;
+};
+
+Display.defaultProps = {
+  result: 'o',
+};
+
```
[the remark this answers is not in the document]
Comment (Torres-ssf) [reply src/components/Display.js]: Screenshot from 2019-12-03 14-11-24

Again the instructions are misleading. This looks way more to an 'o' than a 0

File: src/components/ButtonPanel.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (shloch) [reply src/components/ButtonPanel.js]: please read again, it should render .... `"ALL THE BUTTONS" in the groups. `

It's groups of buttons actually

**Comment by Torres-ssf** [comment]
Hi there @shloch 
Haven't you read my comment in the PR?
The project instructions explicitly require a stateful component for App.js. But the linter complains that I should make it a stateless component instead
Screenshot from 2019-12-03 14-13-57

**Comment by shloch** [comment]
### Hello @Torres-ssf 

Y=Try to follow this feed to deal with that stickler error --> https://stackoverflow.com/questions/43378911/eslint-component-should-be-written-as-a-pure-function-react-prefer-stateless

File: src/components/Display.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (shloch) [reply src/components/Display.js]: hahaha, i see 
it's a ZERO

**Comment by Torres-ssf** [comment]
> ### Hello @Torres-ssf
> Y=Try to follow this feed to deal with that stickler error --> https://stackoverflow.com/questions/43378911/eslint-component-should-be-written-as-a-pure-function-react-prefer-stateless

@shloch Thank you for the help

File: src/components/Display.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ const Display = (props) => {
 };
 
 Display.defaultProps = {
-  result: 'o',
+  result: 0,
 };
 
 Display.propTypes = {
```
Comment (shloch) [inline src/components/Display.js]: this should be `result: '0'` 
Given that the propType is string ....
Please rectify this

**Pull request merged**